CMakeLists.txt

```diff
@@ -44,6 +44,18 @@ include(DefineOptions.cmake)
 include(GNUInstallDirs)
 include(FeatureSummary)
 
+set(DARKTABLE_LIBDIR    "${CMAKE_BINARY_DIR}/lib/darktable")
+set(DARKTABLE_BINDIR    "${CMAKE_BINARY_DIR}/bin")
+set(DARKTABLE_DATADIR   "${CMAKE_BINARY_DIR}/share/darktable") # darktablerc, themes, icons, ... Cannot use ${CMAKE_BINDIR}/data because not everything resides there. Like icons (pixmaps)
+set(DARKTABLE_SHAREDIR  "${DARKTABLE_DATADIR}")
+set(DARKTABLE_LOCALEDIR "${CMAKE_BINARY_DIR}/share/locale")
+
+set(DARKTABLE_RELATIVE_BINDIR    "./${CMAKE_INSTALL_BINDIR}")
+set(DARKTABLE_RELATIVE_LIBDIR    "../${CMAKE_INSTALL_LIBDIR}/darktable")
+set(DARKTABLE_RELATIVE_DATADIR   "../${CMAKE_INSTALL_DATAROOTDIR}/darktable")
+set(DARKTABLE_RELATIVE_SHAREDIR  "${DARKTABLE_RELATIVE_DATADIR}")
+set(DARKTABLE_RELATIVE_LOCALEDIR "../share/locale")
+
```

Comment:
In all likelihood this approach breaks macOS bundle, since directory structure is different in it compared to installed version - binary is MacOS/darktable, while everything else is in Resources/.

Reply:
@parafin  Hmm, I have no Mac. Can you test that?

Reply:
I will when I have time, I can't promise when.

Reply:
I will resolve that because its not on my side anymore.